twitter/twitter.php

```diff
@@ -557,28 +559,24 @@ function twitter_post_hook(App $a, array &$b)
 		}
 
 		$condition = ['uri' => $b["thr-parent"], 'uid' => $b["uid"]];
-		$orig_post = Item::selectFirst(['author-nick', 'author-network'], $condition);
-		if (!DBA::isResult($orig_post)) {
+		$thr_parent = Item::selectFirst(['uri', 'author-nick', 'author-network'], $condition);
```

Comment:
Can you include `extid` to this as well? Seems like it is needed in line 610.

Reply:
Good catch!